cosmos/profiles/base.py

```diff
@@ -41,11 +102,18 @@ class BaseProfileMapping(ABC):
 
     _conn: Connection | None = None
 
-    def __init__(self, conn_id: str, profile_args: dict[str, Any] | None = None, disable_event_tracking: bool = False):
+    def __init__(
+        self,
+        conn_id: str,
+        profile_args: dict[str, Any] | None = None,
+        disable_event_tracking: bool | None = None,
+        dbt_config_vars: DbtProfileConfigVars | None = None,
```

Comment:
I think in the meantime cosmos will need to properly handle the case where `disable_event_tracking=True` and `DbtProfileConfigVars(send_anonymous_usage_stats=True)` and should raise a [CosmosValueError](https://github.com/astronomer/astronomer-cosmos/blob/1b7f4a06e06cb5af75bf411a28dca932ef37ab5d/cosmos/exceptions.py#L4-L5)
This will need a unit test.

Reply:
fix and add test